px4_ros2_cpp/test/integration/global_navigation.cpp

```diff
@@ -122,7 +125,7 @@ TEST_F(GlobalPositionInterfaceTest, fuseAll) {
       break;
     }
 
-    rclcpp::spin_some(_node);
+    _executor.spin_some();
```

Comment:
Can you generally use `spin_node_some` to preserve the existing behavior? The difference is that it will also remove the node again.


```suggestion
    _executor.spin_node_some(_node);
```